.github/workflows/build-and-deploy-gh_pages.yml

```diff
@@ -15,5 +15,5 @@ jobs:
           ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
           TARGET_REPO: KYOSS/website
           TARGET_BRANCH: gh_pages
-          BUILD_SCRIPT: yarn && yarn docs:build
+          BUILD_SCRIPT: npm i && npm run docs:build
```

Comment:
What is `npm i` -- is it an alias for `npm install` or something?  I couldn't find it listed under https://docs.npmjs.com/cli/v8/commands or https://docs.npmjs.com/cli/v8/commands/npm-init.

Reply:
Yes, `npm i` is shorthand for `npm install`. The `npm` commands can be shortened to anything which is unambiguous for the available options.

Reply:
Can't that be confused with `npm init`? 😄 

Can you update the commit to make both instances of this be `npm install` instead of `npm i`?  For those of us not conversant with the intricacies of `npm`, it makes the code easier to read / understand (i.e., even though `npm i` is technically correct, it's friendlier to Future People who have to come read this code).

Thanks!